resource/systemd/daemon_reload.go

```diff
@@ -5,6 +5,9 @@ import (
 	"github.com/coreos/go-systemd/dbus"
 )
 
+//CheckDaemonReload checks whether the systemd daemon needs to be reloaded.
+//it does not check if a unit is failed and needs to be reset though.
+//NOTE : should I implement checkResetFailed?
```

Comment:
I'll be heading out soon, but if this functionality is necessary, I'll implement it.